fe1-web/src/features/home/navigation/MainNavigation.tsx

```diff
@@ -34,15 +35,11 @@ const MainNavigation: React.FC = () => {
         swipeEnabled: false,
       }}>
       <HomeTopTabNavigator.Screen name={STRINGS.navigation_tab_home} component={Home} />
```

Comment:
why not merge `Home` and `Launch` into `screens` within this component, instead of `Home` here, `Launch` in `home/index.ts` and a bit elsewhere ?

Reply:
👍